<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>More</class>
    <widget class="QWidget" name="More">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1000</width>
                <height>618</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>More</string>
        </property>
        <property name="styleSheet">
            <string notr="true">QWidget#More
                {
                background-color:white
                }
            </string>
        </property>
        <widget class="QFrame" name="frame_name">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>128</width>
                    <height>618</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">QFrame#frame_name
                    {
                    background-color:white
                    }
                </string>
            </property>
            <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
                <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="pushButton_about">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>128</width>
                        <height>32</height>
                    </rect>
                </property>
                <property name="contextMenuPolicy">
                    <enum>Qt::CustomContextMenu</enum>
                </property>
                <property name="styleSheet">
                    <string notr="true">QPushButton#pushButton_about
                        {
                        border:0px solid rgb(16,124,16);
                        background-color:white
                        }
                        QPushButton#pushButton_about:hover
                        {
                        border:2px solid rgb(16,124,16);
                        border-color:green
                        }
                        QPushButton#pushButton_about:pressed
                        {
                        background-color:rgb(195, 195, 195)
                        }
                    </string>
                </property>
                <property name="text">
                    <string>关于</string>
                </property>
            </widget>
        </widget>
        <widget class="QFrame" name="frame_content">
            <property name="geometry">
                <rect>
                    <x>128</x>
                    <y>0</y>
                    <width>872</width>
                    <height>618</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">QFrame#frame_content
                    {
                    background-color:white
                    }
                </string>
            </property>
            <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
                <enum>QFrame::Raised</enum>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
